### PR TITLE
Added documentation for debugDump

### PR DESCRIPTION
**Pull request by cpjulia**
arangosh examples modified/added in 3.10, 3.11, 3.12

### CODE REVIEW

**Comment by Simran-B** [comment]
Simplified the content and linked to the existing documentation of `debugDump()`, which I also extended a little bit.

**Comment by Simran-B** [comment]
Example generation for 3.12 for preview purposes:
https://jenkins.arangodb.biz/view/Documentation/job/arangodb-ANY-examples/3301/

File: 3.10/troubleshooting-query-debug-packages.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+contains information about the query and the environment to make it possible to
+reproduce the issue:
+
+- General information about the server including the exact version
+- The properties of the involved database, collections, Views, and graphs
+- The query, its bind variables, and options
+- The query execution plan
+- Storage engine statistics
+- Optionally samples of your data, with the option to
+  **obfuscated all string values** in a non-reversible way
```
Comment (nerpaula) [inline 3.10/troubleshooting-query-debug-packages.md:23]:
```suggestion
  **obfuscate all string values** in a non-reversible way
```

**Comment by Simran-B** [comment]
https://jenkins.arangodb.biz/view/Documentation/job/arangodb-ANY-examples/3306/

**Review by nerpaula — APPROVED** [review]

**Pull request merged**